services/topology-engine/queue-engine/topologylistener/message_utils.py

```diff
@@ -37,8 +37,15 @@
 
 
 class Abstract(model.JsonSerializable):
-    def to_json(self):
-        return json.dumps(self, cls=model.JSONEncoder)
+    def __str__(self):
+        return '<{}: {}>'.format(type(self).__name__, self.to_json(
+            uniformly=True))
+
+    def to_json(self, uniformly=False):
```

Comment:
why not to name this flag as sort_keys and pass it as is to json.dumps?

Reply:
Good question. :) Will change it in proposed way.